src/test/java/com/example/bookstore/BookStoreApplicationTests.java

```diff
@@ -1,13 +1,8 @@
 package com.example.bookstore;
 
-import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
 class BookStoreApplicationTests {
 
-    @Test
-    void contextLoads() {
-    }
-
```

Comment:
I do not see a reason to remove this test, it can actually be useful to know that our application can at least pass some smoke test if we do not have any others. Just wanted to mention it :)